_includes/coinduction-extreme-predicates-ordinals.html

```diff
@@ -0,0 +1,1744 @@
+<html lang="en-US">
```

Comment:
Please remove the html tags, the meta tags (Jekyll generates its own anyway), unwrap the <head> and unwrap the <body>.
That will help Jekyll generate the blog post without weird artefacts